ckanext/switzerland/templates/scheming/form_snippets/publisher.html

```diff
@@ -2,13 +2,13 @@
 {% set publisher = h.ogdch_publisher_form_helper(data) %}
 
 {# Input for publisher names in multiple languages #}
-{% for lang in ['fr', 'de', 'it', 'en'] %}
+{% for lang in h.fluent_form_languages(field, entity_type, object_type, schema) %}
```

Comment:
Making the order of languages the same as in the other multilingual form snippets.